{{cookiecutter.project_dashed}}-src/ansible/group_vars/beta

```diff
@@ -1,4 +1,4 @@
-project_ident_env: {{ cookiecutter.project_namespace }}-beta
+app_environment: BETA
```

Comment:
Any reason why all caps is used?  If not, can we use lowercase?

I don't remember exactly where project_ident_environment would be used, but at least on RaceBetter, that ends up being part of the directory names on shadow.  I'd really not want to see "/home/racebetter-BETA/" or "/var/log/racebetter/racebetter-BETA.app" etc.

Reply:
Done.